lib/monetary.ts

```diff
@@ -0,0 +1,209 @@
+import Big from 'big.js';
+import { BigSource } from 'big.js';
+
+
+export interface ICurrency {
```

Comment:
Typescript doesn't require separate interface names, if a class with the same name inherits this type then the compiler should be able to differentiate.